### DISCUSSION TITLE
Add a link to Discussions in the documentation

### DISCUSSION THREAD

**Discussion by jdhitsolutions** [buttondown/discussions#29] [General]
I would find it very helpful if you would include a link to the GitHub discussions in the documentation. I'm always floundering trying to remember the link. And you might get more interaction if you made it easier for people to find this.

**Comment by jmduke** [comment]
Absolutely! We're actually working on a brand-new docs site and just filed it for inclusion there: https://github.com/buttondown/roadmap/issues/1716
